Source/Dafny/Dafny.atg

```diff
@@ -2608,7 +2608,7 @@ FunctionSpec<.List<AttributedExpression> reqs, List<FrameExpression> reads, List
   .
 
 /*------------------------------------------------------------------------*/
-PredicateResult<string name, out Formal? result>
+PredicateResult<string name, out Formal result>
```

Comment:
I don't think you can remove the question mark, unless now predicates always return a formal ^^

Reply:
The `?` syntax is not enabled for this project, so having the `?` creates a warning. Since the `?` syntax is not enabled, not having it does not imply that it cannot be null.

Reply:
Ok good point thanks.